conf/test.config

```diff
@@ -24,28 +30,25 @@ params {
 
     // Small reference genome
     igenomes_ignore   = true
-    genome            = 'small_hg38'
-    genomes_base      = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules'
+    genome            = null
+
+    dbsnp             = params.test_data['homo_sapiens']['genome']['dbsnp_146_hg38_vcf_gz']
+    fasta             = params.test_data['homo_sapiens']['genome']['genome_fasta']
+    germline_resource = params.test_data['homo_sapiens']['genome']['gnomad_r2_1_1_vcf_gz']
+    intervals         = params.test_data['homo_sapiens']['genome']['genome_interval_list']
+    known_indels      = params.test_data['homo_sapiens']['genome']['mills_and_1000g_indels_vcf_gz']
 
-    dbsnp                   = "${params.genomes_base}/data/genomics/homo_sapiens/genome/vcf/dbsnp_146.hg38.vcf.gz"
-    fasta                   = "${params.genomes_base}/data/genomics/homo_sapiens/genome/genome.fasta"
-    germline_resource       = "${params.genomes_base}/data/genomics/homo_sapiens/genome/vcf/gnomAD.r2.1.1.vcf.gz"
-    intervals               = "${params.genomes_base}/data/genomics/homo_sapiens/genome/genome.interval_list"
-    known_indels            = "${params.genomes_base}/data/genomics/homo_sapiens/genome/vcf/mills_and_1000G.indels.vcf.gz"
     nucleotides_per_second  = 20
 
     snpeff_db         = 'WBcel235.99'
     vep_species       = 'caenorhabditis_elegans'
-    vep_cache_version = '104'
-
-    // Ignore `--input` as otherwise the parameter validation will throw an error
-    schema_ignore_params = 'genomes,input'
```

Comment:
we don't need this anymore?

Reply:
Will verify that once more, now that I see it, I'm wondering the same